### PR TITLE
feat: adds blue green deployments

### PR DESCRIPTION
**Pull request by tdstein**

### CODE REVIEW

**Comment by tdstein** [comment]
Closes https://github.com/posit-dev/connect-cookbook/issues/95

File: 14-mlops/04-blue-green-deployments/index.qmd [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+title: Blue / Green Deployments
+---
+
+## Problem
+
+You want to utilize a blue-green deployment method to safely deploy changes to production content.
+
+## Solution
+
+Use [Vanity URLs API](/api/#get-/v1/content/-guid-/vanity) to switch traffic between deployments. In this example, we assume that *blue* content has an assigned vanity URL. See [Viewing a Content Items Vanity URL](../../04-content/19-viewing-a-content-items-vanity-url/index.qmd).
```
Comment (toph-allen) [inline 14-mlops/04-blue-green-deployments/index.qmd:11]: I'm not sure why, but at least in Safari, on Dogfood, the scroll offset is messed up for the API docs. It leaves me scrolled to the bottom of the page, where the destination is visible in the sidebar, and I have to scroll up to see the actual thing I wanna view.

I guess this should be a bug report filed against the API docs.

Comment (toph-allen) [reply 14-mlops/04-blue-green-deployments/index.qmd:11]: Filed.

**Review by toph-allen — APPROVED** [review]
This is cool. I feel like we almost want a bit of a distinction between the recipes that are like "do a thing" and those which combine the smaller steps into larger example strategies. Just something I'm mulling over. :)

**Pull request merged**